src/SignalR/clients/java/signalr/.gradle/buildOutputCleanup/cache.properties

```diff
@@ -0,0 +1,2 @@
+#Fri Dec 07 17:30:44 PST 2018
+gradle.version=4.9
```

Comment:
It looks like something needs to be added to the gitignore. I don't think we want to commit files in `.gradle/`